### PR TITLE
initial ci for adding footnote function

### PR DESCRIPTION
**Pull request by graceannobrien**

### CODE REVIEW

File: R/get_equations.R [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    .zscore = 1.64
+  } else if (.ci == 90){
+    .zscore = 1.96
+  } else if (!(.ci %in% c(90, 95)) & !is.na(.zscore)){
+    .zscore = .zscore
+  } else {
+    stop("Z-score (.zscore) must be supplied when CI is not 90 or 95")
+  }
+
+  x <- list(
```
Comment (KatherineKayMRG) [inline R/get_equations.R:24]: What do you think to adding...

```
ci   = "CI: confidence intervals"
corr = "Corr: Correlation coefficient"
cv   = "CV: coefficient of variation"
se   = "SE: standard error"
```
We might need to think about "credible intervals" too for the Bayes tables (that's often abbreviated to 95% CrI but lets check with Tim).

File: R/get_equations.R [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    ciEq =  paste0("Confidence intervals = estimate $\\pm$ ", .zscore, " $\\cdot$ SE"),
+    cvOmegaEq = "CV\\% of log-normal omegas = sqrt(exp(estimate) - 1) $\\cdot$ 100",
+    cvSigmaEq = "CV\\% of sigma = sqrt(estimate) $\\cdot$ 100",
+    log = "Parameters estimated in the log-domain were back-transformed for clarity.",
```
Comment (KatherineKayMRG) [inline R/get_equations.R:28]: Can we please update "log" to "logTrans" and "logit" to "logitTrans"?

**Comment by graceannobrien** [comment]
@KatherineKayMRG I'm making example for `get_equations`- I am making all abbreviations on one line, and then equations each get their own line. What do you think about this work flow?
```
footnotes = get_equations()
make_pmtable(.df = data, .pmtype = "random") %>%
  pmtables::st_notes(paste0("Abbreviations: ",
                     footnotes$ci),
                     footnotes$cv,
                     collapse= "; ",
                     to_string = TRUE) %>%
  pmtables::st_notes(footnotes$cvOmegaEq,
                     footnotes$cvSigmaEq,
                     collapse= "; ") %>%
  pmtables::stable() %>%
  pmtables::st2report()
```
Screenshot 2024-05-13 at 9 40 40 AM

**Comment by KatherineKayMRG** [comment]
@graceannobrien - I'll try to look closer at this soon. My first thought is that we need to drop `Abbreviations: ` because that doesn't match the style in pmtables. Also, this is just an example for people to see how it could be used right? You're not proposing making the `st_notes` part of the regular package workflow right? 

I would also update 
```
pmtables::st_notes(footnotes$ci), footnotes$cv) %>%
st_notes_str() %>%
pmtables::st_notes(footnotes$cvOmegaEq,
                     footnotes$cvSigmaEq) %>%
````

**Comment by graceannobrien** [comment]
> @graceannobrien - I'll try to look closer at this soon. My first thought is that we need to drop `Abbreviations: ` because that doesn't match the style in pmtables. Also, this is just an example for people to see how it could be used right? You're not proposing making the `st_notes` part of the regular package workflow right?
> 
> I would also update
> 
> ```
> pmtables::st_notes(footnotes$ci), footnotes$cv) %>%
> st_notes_str() %>%
> pmtables::st_notes(footnotes$cvOmegaEq,
>                      footnotes$cvSigmaEq) %>%
> ```

Yes, this is just an example of how it would be used.

**Comment by KatherineKayMRG** [comment]
ok thanks @graceannobrien - I would still recommend updating to the code without the paste

File: vignettes/getting-started.Rmd [hunk 3]
```diff
@@ -108,19 +108,18 @@ We leverage `make_pmtable` to generate fixed effects and random effects paramete
...
+footnote <- get_equations(.ci = 95)
 ```
 
 ```{r}
 ##fixed
-tableList$fixed <- make_pmtable(df2, .pmtype = "fixed", .notes = c(footNote1, footNote2))
+tableList$fixed <- make_pmtable(df2, .pmtype = "fixed") %>% 
+  #abbreviations
+  st_notes(footnote$ci, footnote$se) %>% 
+  #equations
```
Comment (KatherineKayMRG) [inline vignettes/getting-started.Rmd:120]: really minor thing but the next line is collapsing the abbreviation footers to a single line (it's not related to the equations. Maybe worth moving the comment (here and below)

File: vignettes/making-bootstrap-parameter-tables.Rmd [hunk 4]
```diff
@@ -129,11 +129,16 @@ tableList$bootstrap <- df2 %>%
   mutate(boot_ci_95 = if_else(is.na(boot_ci_95), "FIXED", boot_ci_95)) %>% 
   st_new() %>% 
   st_panel("desc") %>% 
+  st_rename(Value = boot_value,
+            "95% CI" = boot_ci_95,
+            " "= abb) %>% 
+  #abbreviations
   st_notes(footnote$ci) %>% 
   st_notes_str() %>%
```
Comment (KatherineKayMRG) [inline vignettes/making-bootstrap-parameter-tables.Rmd:137]: If I'm reading this right and you only have one abbreviation in that line 136 footnote then there is no need to collapse it to a string in line 137

**Review by andersone1 — APPROVED** [review]

File: R/get_equations.R [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        1.96
+      } else {
+        .zscore
+      }
+    }
+
+
+  list(
+    ci   = "CI: confidence intervals",
+    corr = "Corr: Correlation coefficient",
```
Comment (KatherineKayMRG) [inline R/get_equations.R:52]: lower case c on correlation
```suggestion
    corr = "Corr: correlation coefficient",
```

File: _pkgdown.yml [hunk 6]
```diff
@@ -33,3 +33,7 @@ reference:
     desc: Generates specific parameter tables by filtering and using pmtables.
     contents:
     - make_pmtable
+  - title: Get generic paramter table footnotes
```
Comment (KatherineKayMRG) [inline _pkgdown.yml:36]: typo
```suggestion
  - title: Get generic parameter table footnotes
```

File: NAMESPACE [hunk 7]
```diff
@@ -4,6 +4,7 @@ export(define_boot_table)
 export(define_param_table)
 export(format_boot_table)
 export(format_param_table)
+export(get_equations)
```
Comment (KatherineKayMRG) [inline NAMESPACE:7]: @graceannobrien I know it's late to ask but could we update the name of this function? It does more than return equations. What about `get_param_notes` or just `param_notes`?

**Review by KatherineKayMRG — APPROVED** [review]
Thanks @graceannobrien

**Pull request merged**